test/functional/uuid/jsonb.ts

```diff
@@ -7,10 +7,17 @@ import {createTestingConnections, closeTestingConnections} from "../../utils/tes
 describe("uuid type", () => {
 
     let connections: Connection[];
-    before(async () => connections = await createTestingConnections({
-        entities: [Record],
-        enabledDrivers: ["postgres"] // because only postgres supports jsonb type
-    }));
+    before(async () => {
+        connections = await createTestingConnections({
+            entities: [Record],
+            enabledDrivers: ["postgres"] // because only postgres supports jsonb type
+        });
+
+        connections.forEach(async connection => {
+            let queryRunner = await connection.driver.createQueryRunner();
+            await queryRunner.query(`CREATE extension IF NOT EXISTS "uuid-ossp"`);
```

Comment:
another question is, why need to execute this statement, do user always need to do this before he is going to use uuid? Isnt there a way not to do this and use uuid?